roles/analysis/tasks/main.yml

```diff
@@ -15,4 +21,18 @@
     mode: "0644"
     owner: root
     group: root
+
+- name: Include tasks for convert2rhel pre-conversion analysis
+  ansible.builtin.include_tasks: analysis-c2r.yml
+  # NOTE: I guess this should be extended to more distros
+  when:
+    - ansible_distribution == "CentOS" and ansible_distribution_major_version|int == 7
```

Comment:
@r0x0d can I remove this? will the execution fail correctly in common role?

Reply:
Yeah, I think we can safely remove it... We already have this check in the common, so, if that fails we can fix it in there.